ports/onnxruntime/portfile.cmake

```
@@ -0,0 +1,328 @@
+# Only limited triplets supported
+# x64-windows: x64-windows-static-md
+# x64-linux
+# No support for osx and uwp OS
+if(VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_fail_port_install(ON_LIBRARY_LINKAGE "dynamic")
+    vcpkg_fail_port_install(ON_TARGET "uwp")
+    vcpkg_fail_port_install(ON_ARCH "arm" "arm64" "wasm32" "x86")
+elseif(VCPKG_TARGET_IS_LINUX)
+    vcpkg_fail_port_install(ON_ARCH "x86")
+else()
+    vcpkg_fail_port_install(ALWAYS)
+endif()
```

```suggestion
    vcpkg_fail_port_install(ON_ARCH "arm" "arm64" "wasm32" "x86" ON_TARGET "uwp")
    
    vcpkg_check_linkage(ON_STATIC_LIBRARY)
```
If it only supports static build ,we can use `vcpkg_check_linkage()`.

---

Thank yo. I will make the above code succinct as you suggest 👍